Scenes/Player/Player.cs

```diff
@@ -126,11 +126,12 @@ public partial class Player : CharacterBody2D {
             AddChild(Sword);
             Sword.Rotation = GetAngleToMouse().NearestDirection8().Radians();
         }
-        if (Sword is Sword sword && sword.Lifetime >= MeleeDuratio   {
+
+        if (Sword is not null && Sword.Lifetime >= MeleeDuration) {
```

Comment:
too much `Sword`? :(

I really prefer the style where `sword` is a proven non-null local variable and we use that for method calls. No risk of NPE

Reply:
But if `Sword is null`, it short circuits, right?